src/components/molecules/UiPhoneNumber/UiPhoneNumber.vue

```diff
@@ -105,6 +106,10 @@ const modelValue = computed({
   get: () => props.modelValue,
   set: (value) => emit('update:modelValue', value),
 });
+const attrs:PhoneNumberAttrsProps = useAttrs();
+const hasError = computed(() => ((!attrs.class)
+  ? false
+  : attrs.class.includes('ui-phone-number--has-error')));
```

Comment:
I've heard that it's a good practice _to avoid negations_ in conditions whenever it's possible, whenever it makes sense, or whenever it doesn't obfuscate the meaning of code. 
Wouldn't it be easier to read with the opposite condition? Or is there a reason for the _negation_ in the first condition?
WDYT:
```suggestion
const hasError = computed(() => ((attrs.class)
  ? attrs.class.includes('ui-phone-number--has-error')
  : false));
```

I've also met the airbnb javascript styleguide, which includes many good patterns. One of them is _comparison shortcuts_, which suggest to **_Use shortcuts for booleans, but explicit comparisons for strings and numbers_** I think it could be extended to **Use shortcuts for booleans, but explicit comparisons for other types**. Would you agree?
According to that, maybe it'd be better to change `(attrs.class) ?` to an explicit comparison. :) WDYT?

Reply:
@AlanLes may I utilize 'explicit comparison' to check if the object has the key 'class'? As far as I know, I can use 'class' in attrs or attrs.hasOwnProperty('class'). This approach aligns with the guideline "Use shortcuts for booleans".